vendor/modules.txt

```diff
@@ -347,33 +347,34 @@ k8s.io/apimachinery/pkg/util/runtime
 k8s.io/apimachinery/pkg/watch
```

Comment:
please move vendoring into its own commit

Reply:
also this is missing packages to be vendored /go/src/github.com/openshift/machine-api-operator/vendor/k8s.io/client-go/dynamic/fake, please include them via
go mod tidy, go mod vendor

Reply:
https://github.com/openshift/machine-api-operator/pull/472 has been created to handling vendoring for k8s.io/client-go/dynamic/fake. Will mark this PR as WIP until the vendoring merges.

Reply:
please include any vendoring required by this PR in this PR